mix.exs

```diff
@@ -5,7 +5,8 @@ defmodule Inky.MixProject do
     [
       app: :inky,
       version: "1.0.2",
-      elixir: "~> 1.8",
+      elixir: "~> 1.9",
```

Comment:
I'm actually using Elixir 1.15, I was just trying to be minimal on bumping the minimum version. Although I think I'll bump up to 1.11 so that we can use `Logger.warning` instead of the deprecated `Logger.warn`